docs/application/tizen-studio/setup/install-ubuntu.md

```diff
@@ -0,0 +1,181 @@
+> - Alternatively, you can also try using Oracle JDK 10 or less instead of OpenJDK, see the official site for the installation steps.
+
+## Setup OpenJDK in Ubuntu
+
+Following steps help you to setup OpenJDK: 
+
+1. Extract the downloaded **openjdk-10_linux-x64_bin.tar.gz**, and **javafx-11-0-2-sdk-linux.zip** files.
+   - Locate **jdk-10** and **javafx-sdk-11.0.2** directories in your file system.
+   > **Note:**
+   >
+   >  jdk-10, javafx-sdk-11.0.2 directories get created as a result of extraction. 
+   
+2. Navigate to the **lib** folder in the **javafx-sdk-11.0.2** directory and copy all the files except **src.zip**.
+3. Navigate to the **lib** folder in the **jdk-10** directory and paste all the copied files.
+
+Following steps help you to set up the JDK path:  
+
+1.	Open the Terminal.
+2.	Type the following command in the terminal to set the JAVA_HOME as OpenJDK directory path, for example:
+	JAVA_HOME=/home/<username>/Desktop/openJDK/jdk-10.0.2/
+	
+3.	Type the following commands in the terminal one after the other:
+	- `sudo update-alternatives --install /usr/bin/java java ${JAVA_HOME%*/}/bin/java 20000`
+	- `sudo update-alternatives --install /usr/bin/javac javac ${JAVA_HOME%*/}/bin/javac 20000`
+	
+4.	Type the following command to get a list of JDK versions installed:
+	- `sudo update-alternatives --config java`
+	
+	> **Note:**
+    >
+    >  If there are multiple JDK installed, enter the selection number to choose your desired JDK version.
+	
+5.	Type the following command:
+	- `sudo update-alternatives --config javac`
+	
+	> **Note:**
+    >
+    >  javac is used for compilation. For more information on the differences between java and javac, see the official [site](https://docs.oracle.com).
+	
+6.	Type the following command to verify whether the OpenJDK version 10 is installed:
+	`java –version`
+
+## Install Dependencies
+
+The Tizen Studio application may require additional libraries in order to work properly. The installer already provides the immediate libraries needed. Various distributions differ on how they are bundled; however, users have to manually install additional packages in order to have error free installation experience:
+
+Open your Terminal, and execute the following commands: 
+- `sudo apt-get install expect\`
+- `sudo apt-get install gtk2-engines-pixbuf\`
+- `sudo apt-get install libgnome2-0\`
+- `sudo apt-get install qemu-user-static\`
+- `sudo apt-get install libwebkitgtk-1.0-0\`
+- `sudo apt-get install gettext\`
+- `sudo apt-get install gksudo\`
+- `sudo apt-get install module-init-tools\`
+- `sudo apt-get install libudev-dev\`
+- `sudo apt-get install libsdl1.2debian\`
+- `sudo apt-get install bridge-utils\`
+- `sudo apt-get install openvpn\`
+
+## Launch Installer 
+Following steps help you to launch the installer:
+
+1. Open your terminal and execute the following command:
+	- `chmod +x web-ide_Tizen_Studio_3.1_ubuntu-64.bin`
```

Comment:
If you want to write the version information please try to follow up the latest version always. For now, Tizen Studio 3.2 is the latest